x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -129,6 +137,9 @@ Show the current {agent} policy.
 
 If no parameters are specified, shows the full {agent} policy.
 
+//REVIEWERS: The CLI help still says "configuration file." I think we need to
+//change this to say policy file, no?
```

Comment:
Hmm, thats a good question...

Configuration file is used on almost all software. I don't know if we should change it when it comes to changing how Elastic Agent operates locally.

Reply:
Changing back to say "configuration" when talking about the local config file.